nnotepad/TODO.md

```diff
@@ -0,0 +1,17 @@
+# To-Do
+
+## Basics
+
+* Style to match rest of webnn-samples.
+* Improve default text.
+* Consider incorporating [WebNN Polyfill](https://github.com/webmachinelearning/webnn-polyfill).
+* Make input/output areas resizable.
+* Add to `../README.md` once we're happy with it.
+
```

Comment:
- Style to match rest of webnn-samples.
 - Consider incorporating [WebNN Polyfill](https://github.com/webmachinelearning/webnn-polyfill).

I can take these two TODOs.